<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { 
background: #d9d9d9;
color : #222
}
QTabWidget::pane {
border: none;
background: none;
}
QTabWidget::tab-bar {
top : 0;
left : 0;
}

QTabBar::tab {
border: none;
padding :0 0 15px;
width : 80px;
height :65px;
background: #fff;
}
QTabBar::tab:selected {}

QTabBar::tab:!selected {}

QPushButton {
background : #428dff;
color : #fff;
border-radius : 5
}
QPushButton:disabled{
background : #5c9bf9;
} 
QPushButton:hover {
background : #fff;
color :#428dff
}

QTextEdit , QLineEdit{
background : #fff;
color :#333;
border-radius : 8;
padding : 5;
border: none
}
QTextEdit:focus, QLineEdit:focus{
border: 1px solid #999;
}

QTableWidget{
background : #fff;
border:none;
border-radius : 8;
}
QHeaderView::section, QTableWidget::Item{
    background: rgba(0,0,0,0);
    font-size: 14pt;
    border-style: none;
    border-bottom: 0.5 solid #ddd;
    border-right: .5 solid #ddd;
}




QScrollBar:horizontal {
    border: 1px solid #222222;
    background: QLinearGradient(x1:0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0.0 #121212,
    stop: 0.2 #282828,
    stop: 1 #484848);
    height: 7px;
    margin: 0px 16px 0 16px;
}

QScrollBar::handle:horizontal {
    background: QLinearGradient(x1:0,
    y1: 0,
    x2: 1,
    y2: 0,
    stop: 0 #ffa02f,
    stop: 0.5 #d7801a,
    stop: 1 #ffa02f);
    min-height: 20px;
    border-radius: 2px;
}

QScrollBar::add-line:horizontal {
    border: 1px solid #1b1b19;
    border-radius: 2px;
    background: QLinearGradient(x1:0,
    y1: 0,
    x2: 1,
    y2: 0,
    stop: 0 #ffa02f,
    stop: 1 #d7801a);
    width: 14px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
    border: 1px solid #1b1b19;
    border-radius: 2px;
    background: QLinearGradient(x1:0,
    y1: 0,
    x2: 1,
    y2: 0,
    stop: 0 #ffa02f,
    stop: 1 #d7801a);
    width: 14px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::right-arrow:horizontal, QScrollBar::left-arrow:horizontal {
    border: 1px solid black;
    width: 1px;
    height: 1px;
    background: white;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}

QScrollBar:vertical {
    background: QLinearGradient(x1:0,
    y1: 0,
    x2: 1,
    y2: 0,
    stop: 0.0 #121212,
    stop: 0.2 #282828,
    stop: 1 #484848);
    width: 7px;
    margin: 16px 0 16px 0;
    border: 1px solid #222222;
}

QScrollBar::handle:vertical {
    background: #428dff;
    min-height: 20px;
    border-radius: 2px;
}

QScrollBar::add-line:vertical {
    border: 1px solid #1b1b19;
    border-radius: 2px;
    background:#428dff;
    height: 14px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    border: 1px solid #1b1b19;
    border-radius: 2px;
    background: QLinearGradient(x1:0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0 #d7801a,
    stop: 1 #ffa02f);
    height: 14px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    border: 1px solid black;
    width: 1px;
    height: 1px;
    background: white;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}





QComboBox {
    selection-background-color: #ffaa00;
    background-color: QLinearGradient(x1:0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0 #565656,
    stop: 0.1 #525252,
    stop: 0.5 #4e4e4e,
    stop: 0.9 #4a4a4a,
    stop: 1 #464646);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
}

QComboBox:hover, QPushButton:hover {
    border: 2px solid QLinearGradient(x1:0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0 #ffa02f,
    stop: 1 #d7801a);
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 4px;
    background-color: QLinearGradient(x1:0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0 #2d2d2d,
    stop: 0.1 #2b2b2b,
    stop: 0.5 #292929,
    stop: 0.9 #282828,
    stop: 1 #252525);
    selection-background-color: #ffaa00;
}

QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color: QLinearGradient(x1:0,
    y1: 0,
    x2: 0,
    y2: 1,
    stop: 0 #ffa02f,
    stop: 1 #d7801a);
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/down_arrow.png);
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>700</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tap_home">
    <attribute name="icon">
     <iconset resource="imgqrc.qrc">
      <normaloff>:/images/home.png</normaloff>
      <activeon>:/images/home-color.png</activeon>:/images/home.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QComboBox" name="in_type">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JOD</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>the last 30 activty from last month</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="in_note">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>531</width>
       <height>71</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Do u have any notes ?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_addnew">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>add new activity</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="in_value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>531</width>
       <height>501</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last edit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vlaue</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Notes</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tap_export">
    <attribute name="icon">
     <iconset resource="imgqrc.qrc">
      <normaloff>:/images/export.png</normaloff>
      <activeon>:/images/export-color.png</activeon>:/images/export.png</iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QTableWidget" name="TB_export">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>541</width>
       <height>481</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last edit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vlaue</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Notes</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_export">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>650</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_brows">
     <property name="geometry">
      <rect>
       <x>502</x>
       <y>610</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="imgqrc.qrc">
       <normaloff>:/images/folder-color.png</normaloff>
       <activeon>:/images/folder-color.png</activeon>:/images/folder-color.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_location">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>file name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="in_type_export">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JOD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All types</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="in_time">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Last week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last 3 month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last 6 month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last year</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All time</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>select by date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>select by type</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame { 
background : '#fff';
border : none
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources>
  <include location="imgqrc.qrc"/>
 </resources>
 <connections/>
</ui>
